.ci_support/linux_64_openssl1.1.1.yaml

```diff
@@ -25,25 +25,9 @@ ncurses:
 openssl:
 - 1.1.1
 pin_run_as_build:
-  bzip2:
-    max_pin: x
```

Comment:
Do we know why the re-rendering is changing these?

Reply:
Check the global pinnings file. I think @hmaarrfk has been working on removing them since they are an old way of doing things.

Reply:
Yeah I was looking. `git blame` doesn't work as well for missing lines 😄 In any event, if this is expected, great!

Reply:
yes, these removals are expected. beckermr has the history correct.